src/register/tests/RegistrationPage.test.jsx

```diff
@@ -242,6 +242,19 @@ describe('RegistrationPage', () => {
       expect(store.dispatch).toHaveBeenCalledWith(fetchRealtimeValidations(formPayload));
     });
 
+    it('should validate the did you mean suggestions', () => {
+      const registrationPage = mount(reduxWrapper(<IntlRegistrationPage {...props} />));
+
+      registrationPage.find('input#email').simulate('blur', { target: { value: 'test@gmail.con', name: 'email' } });
+      expect(registrationPage.find('RegistrationPage').state('suggestedTopLevelDomain')).toEqual('Did you mean: test@gmail.com?');
+
+      registrationPage.find('input#email').simulate('blur', { target: { value: 'test@fmail.com', name: 'email' } });
+      expect(registrationPage.find('RegistrationPage').state('suggestedServiceLevelDomain')).toEqual('Did you mean: test@gmail.com?');
+
+      registrationPage.find('input#email').simulate('blur', { target: { value: 'test@gmail.com', name: 'email' } });
```

Comment:
Can we test other domains? instead of testing for gmail only for all three cases.